src/runtime/tests/refiner-test.ts

```diff
@@ -143,10 +143,6 @@ describe('refiner', () => {
       entityClass = Entity.createEntityClass(schema, null);
     });
 
-    it('data does not conform to the refinement', Flags.whileEnforcingRefinements(async () => {
```

Comment:
I think we can keep this test. There's no exception thrown any more but we can register a system exception handler and then check that it was called and what it was called with.

Reply:
Just a rough guess at what would be needed in the test body (haven't tested it or checked it).

```      
      let exceptions: Error[] = [];

      removeSystemExceptionHandler(defaultSystemExceptionHandler);
      registerSystemExceptionHandler((arc, exception) => {exceptions.push(exception)});

      const e = new entityClass({txt: 'abc', num: 56});
      assert.lengthOf(exceptions, 1);
      exceptions.map(except => {
         assert.deepEqual(except.message, `Entity schema field 'num' does not conform to the refinement.`);
      });
```